lib/jekyll-optional-front-matter/generator.rb

```diff
@@ -42,9 +42,15 @@ def page_from_static_file(static_file)
 
     # Does the given Jekyll::Page match our filename blacklist?
     def blacklisted?(page)
+      return false if whitelisted?(page)
       FILENAME_BLACKLIST.include?(page.basename.upcase)
     end
 
+    def whitelisted?(page)
+      return false unless site.config["include"].is_a? Array
+      site.config["include"].include?(page.path)
```

Comment:
Thoughts about using Jekyll::EntryFilter here? It does this inclusion stuff already. Maybe overkill.